test/fuzz/gen.ml

```diff
@@ -88,11 +93,16 @@ let expr_available_1_i64 env =
   [ pair B.iunop_64 (const [ S.Nothing ])
   ; pair B.itestop_64 (const [ S.Pop; S.Push (Num_type I32) ])
   ; pair B.i32_wrap_i64 (const [ S.Pop; S.Push (Num_type I32) ])
-  ; pair B.extend_64_i64 (const [ S.Nothing ])
   ; pair B.f32_convert_i64 (const [ S.Pop; S.Push (Num_type F32) ])
   ; pair B.f64_convert_i64 (const [ S.Pop; S.Push (Num_type F64) ])
   ; pair B.f64_reinterpret_i64 (const [ S.Pop; S.Push (Num_type F64) ])
   ]
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.iunop_popcnt_64 (const [ S.Nothing ]) ]
+      else [] )
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.extend_64_i64 (const [ S.Nothing ]) ]
+      else [] )
```

Comment:
Could you put all of them as before and let the fuzzer fail if they're not implemented ? We'll implemented them later.